resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadExternalLibrary.js

```diff
@@ -46,7 +46,7 @@
 	 * @returns {jQuery} Returns a jQuery promise
 	 * @see https://developers.facebook.com/docs/javascript/quickstart/v2.2
 	 */
-	$.loadFacebookAPI = function (callback) {
+	$.loadFacebookSDK = function (callback) {
```

Comment:
I would move it to the `done` method [here](https://github.com/Wikia/app/blob/SOC-261-2/resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadExternalLibrary.js#L56) so that it gets called every time we load the library. This will have a better chance of accounting for `.sso-login` elements that weren't previously on the page if the library is being called for a second time. Again, it will change with SOC-240, but for now I'd say this would be safer.